ui/app/serializers/mfa-login-enforcement.js

```diff
@@ -34,12 +34,13 @@ export default class MfaLoginEnforcementSerializer extends ApplicationSerializer
     }
     return payload;
   }
-  serialize() {
+  serialize(snapshot) {
```

Comment:
`snapshot` is used on line 41-42, because the serialized value `json` was empty for those arrays 😕 I can investigate this again, but I assumed when I made this change that something changed in the ember-data `serializer`. I'll check PKI as well, since there might not have been a test that caught that one

Reply:
Ahh okay. For added context, I remember that when those values _were_ serialized, the keys were removed all together so checking that the key existed on the json was sufficient, because if it didn't exist then it meant it was empty. 

If the `json` value exists and is empty for those arrays now, then that means we can probably remove this code all together as it sounds like the problem may have been fixed on an ember data level ?!

Reply:
ohh interesting -- turns out the component was using `addObject` on the model attribute which is supposed to be an array, but using that method instead of setting it to something directly (`model.thing = thing`) doesn't trigger an update to the snapshot that gets passed into the serializer 😱 Looking at PKI, the component *does* set it directly, so I'm updating the MFA component to set it like that instead.